.devcontainer/install-dagger.sh

```diff
@@ -9,4 +9,13 @@ echo "autoload -U compinit" >> $HOME/.zshrc
 echo "compinit -i" >> $HOME/.zshrc
 echo "sudo chmod a+rwx /var/run/docker.sock" >> $HOME/.zshrc
 $HOME/.local/bin/dagger completion zsh > /usr/share/zsh/site-functions/_dagger
+echo "[alias]" >> $HOME/.gitconfig
```

Comment:
this is your own setup ? :)

Reply:
This is to allow better workshop exprerience